packages/integrations/cloudflare/src/util.ts

```diff
@@ -0,0 +1,16 @@
+export function getProcessEnvProxy() {
+	return new Proxy(
+		{},
+		{
+			get: (target, prop) => {
+				console.warn(
+					// NOTE: \0 prevents Vite replacement
```

Comment:
WHAT! I totally missed this. That's very smart.

Reply:
It's slightly leaning towards a hack that should be fixed in Vite 4 (hopefully) 😬